src/mongo/MongoScrapbook.ts

```diff
@@ -240,6 +240,11 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 			const nonStringLiterals = [mongoParser.mongoParser.NullLiteral, mongoParser.mongoParser.BooleanLiteral, mongoParser.mongoParser.NumericLiteral];
 			if (tokenType === mongoParser.mongoParser.StringLiteral) {
 				parsedObject = stripQuotes(text);
+			} else if (tokenType === mongoParser.mongoParser.RegexLiteral) {
+				let params: string[] = child.text.split('/');
```

Comment:
won't work for escaped '/'s

Reply:
You should have the parser do this processing for you...  Have a specific token or parse element for just the portion inside the slashes.